app/allocation/fields/special_vehicle_check.js

```diff
@@ -0,0 +1,23 @@
+const specialVehicleCheck = {
```

Comment:
Could we use kebabcase in the filename please. Keeps it consistent with the rest of the codebase

Reply:
👍